api/services/RedisClient.js

```diff
@@ -1,9 +1,10 @@
-import redis from 'redis'
-Promise.promisifyAll(redis.RedisClient.prototype)
-Promise.promisifyAll(redis.Multi.prototype)
```

Comment:
Are we being forced to update promises/.then to async/await or is that by choice?

Reply:
by choice but also because I wanted features from the latest version of reds which switches from callbacks to async/await, which is sooo much better